HouseRules_Essentials/Rules/AbilityBackstabAdjustedRule.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace HouseRules.Essentials.Rules
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Boardgame;
+    using Boardgame.BoardEntities.Abilities;
+    using HouseRules.Types;
+    using UnityEngine;
+
+    public sealed class AbilityBackstabAdjustedRule : Rule, IConfigWritable<Dictionary<string, bool>>, IMultiplayerSafe
+    {
+        public override string Description => "Ability AP costs are adjusted";
+
+        private readonly Dictionary<string, bool> _adjustments;
```

Comment:
QQ: Is there a reason for accepting `string` instead of `AbilityKey`?  I feel like there was talk about this in the past but I can't recall.